core/src/rpc/watchtower.rs

```diff
@@ -12,6 +12,76 @@ impl ClementineWatchtower for Watchtower {
         &self,
         _request: Request<Empty>,
     ) -> Result<Response<WatchtowerParams>, Status> {
-        todo!()
+        let winternitz_pubkeys: Vec<WinternitzPubkey> = self
+            .get_winternitz_public_keys()
+            .await?
+            .iter()
+            .enumerate()
+            .flat_map(|(operator_index, pks)| {
+                pks.iter()
+                    .enumerate()
+                    .map(|(timetx_index, pks)| {
+                        let digit_pubkey = pks.iter().map(|m| m.to_vec()).collect();
+
+                        WinternitzPubkey {
+                            d: operator_index as u32,
+                            n0: timetx_index as u32,
```

Comment:
These are incorrect, d and n0 should be the digit and message length, BUT we can get rid of these d and n0 in the proto file as well, since the receiver will know them

Reply:
Fixed in https://github.com/chainwayxyz/clementine/pull/373/commits/c6dc6aa410014a945d8cc80d877309a59866b979